cmd/fortifyExecuteScan_test.go

```diff
@@ -610,3 +610,57 @@ func TestAutoresolveClasspath(t *testing.T) {
 		assert.Equal(t, "some.jar;someother.jar", result, "Expected different result")
 	})
 }
+
+func TestPopulateMavenTranslate(t *testing.T) {
+	t.Run("src without translate", func(t *testing.T) {
+		config := fortifyExecuteScanOptions{Src: "./**/*"}
+		translate, err := populateMavenTranslate(&config, "")
+		assert.NoError(t, err)
+		assert.Equal(t, `[{"classpath":"","src":"./**/*"}]`, translate, "Expected different parameters")
+	})
+
+	t.Run("exclude without translate", func(t *testing.T) {
+		config := fortifyExecuteScanOptions{Exclude: "./**/*"}
+		translate, err := populateMavenTranslate(&config, "")
+		assert.NoError(t, err)
+		assert.Equal(t, `[{"classpath":"","exclude":"./**/*"}]`, translate, "Expected different parameters")
+	})
+
+	t.Run("with translate", func(t *testing.T) {
+		config := fortifyExecuteScanOptions{Translate: `[{"classpath":""}]`, Src: "./**/*", Exclude: "./**/*"}
+		translate, err := populateMavenTranslate(&config, "")
```

Comment:
```suggestion
		translate, err := populateMavenTranslate(&config, "ignored/path")
```

... otherwise test doesn't make clear that class-path parameter is also ignored.